backend/audit/test_etl.py

```diff
@@ -13,6 +14,8 @@
     FindingText,
     CapText,
     SecondaryAuditor,
+    AdditionalUei,
```

Comment:
How about using `AdditionalUeis` and `AdditionalEins` to be consistent with the existing naming in the code